autosubmit/autosubmit.py

```diff
@@ -2892,7 +2899,7 @@ def statistics(expid, filter_type, filter_period, file_format, section_summary,
                                                                                                    job_list.get_job_list()])
             queue_time_fixes = {}
             if job_to_package:
-                current_table_structure = get_structure(expid, BasicConfig.STRUCTURES_DIR)
+                current_table_structure = get_structure(expid, Path(BasicConfig.STRUCTURES_DIR))
```

Comment:
Tried to cover this change, which is pretty simple, but I couldn't reach this part of the code. Dani can confirm later, but it looks like during the memory profiling changes, we removed the `.name` from job packages.

Everywhere that `job_package_persistence` would call `save`, which would get to this line in the code, is now protected by a `hasattr(obj, 'name')` which is `False` for all the job packages we have. This will be left without tests :+1: